public/app/percona/shared/components/Form/FieldAdapters/FieldAdapters.tsx

```diff
@@ -1,20 +1,54 @@
-import React, { FC } from 'react';
+// @ts-nocheck
```

Comment:
No need to add Input and Textarea adapters, use the ones in platform-core please, is a simple refactor, I did the same for DBaaS. Changes to this file can be removed afterwards.

Reply:
Done, removed both